src/bun.js/webcore/response.zig

```diff
@@ -1503,6 +1503,11 @@ pub const Fetch = struct {
             fetch_tasklet.http.?.client.disable_decompression = fetch_options.disable_decompression;
             fetch_tasklet.http.?.client.reject_unauthorized = fetch_options.reject_unauthorized;
 
+            fetch_tasklet.http.?.client.custom_tls_props.setPassphrase(fetch_options.passphrase);
```

Comment:
maybe call it .tls instead of .custom_tls_props?

Reply:
i disagree, tls is very ambgious, `tls_props`?